serve-web/src/Service/OrderService.php

```diff
@@ -119,11 +119,11 @@ public function upsert(
             $order = new $orderClass($client, $madeAt, $issuedAt, $orderNumber);
             $this->em->persist($order);
             $this->em->persist($client);
-            $this->em->flush();
-        } else {
```

Comment:
Just checking you were meant to get rid of the else statement here, and not just the `flush()`s

Reply:
I thought I'd pushed the wrong thing when I first looked at the diff too - there is an `if` there; its just the way git has shown the changes:

```php
        if (!$order) {
            $order = new $orderClass($client, $madeAt, $issuedAt, $orderNumber);
            $this->em->persist($order);
            $this->em->persist($client);
        }

        if ($order->getOrderNumber() !== $orderNumber){
            $order->setOrderNumber($orderNumber);
            $this->em->persist($order);
        }
```

Reply:
And I misread your message 😂  time to log off! Yeah, it was intentional - I wanted to put some logic around it so we aren't updating every order number - just when it changes.

Reply:
Ah must have missed it. Good to go then.